plugins/event_notification/lib/kEventNotificationFlowManager.php

```diff
@@ -151,15 +151,7 @@ public function shouldConsumeEvent(KalturaEvent $event)
 			
 			$scope->resetDynamicValues();
 			
-			$notificationParameters = $notificationTemplate->getContentParameters();
-			foreach($notificationParameters as $notificationParameter)
-			{
-				/* @var $notificationParameter kEventNotificationParameter */
-				if(!is_null($notificationParameter->getValue()))
-					$scope->addDynamicValue($notificationParameter->getKey(), $notificationParameter->getValue());
```

Comment:
I am not skipping the setting of the dynamic values I am simply getting both arrays (getContentParameters & getUserParameters) together and than executing the foreach on both them instead of one by one.

Do you mean that the order is important i.e I first should add run on contentParameters and only than on userParameters?